ndcube/tests/conftest.py

```diff
@@ -13,36 +13,39 @@
 
 
 @pytest.fixture
-def wcs_4d():
+def wcs_4d_t_l_lt_ln():
     header = {
-        'CTYPE4': 'HPLN-TAN',
-        'CUNIT4': 'arcsec',
-        'CDELT4': 5,
-        'CRPIX4': 5,
-        'CRVAL4': 0,
+        'CTYPE1': 'TIME    ',
+        'CUNIT1': 'min',
+        'CDELT1': 0.4,
+        'CRPIX1': 0,
+        'CRVAL1': 0,
+
+        'CTYPE2': 'WAVE    ',
+        'CUNIT2': 'Angstrom',
+        'CDELT2': 0.2,
+        'CRPIX2': 0,
+        'CRVAL2': 0,
 
         'CTYPE3': 'HPLT-TAN',
         'CUNIT3': 'arcsec',
         'CDELT3': 20,
         'CRPIX3': 0,
         'CRVAL3': 0,
 
-        'CTYPE2': 'WAVE    ',
-        'CUNIT2': 'Angstrom',
-        'CDELT2': 0.2,
-        'CRPIX2': 0,
-        'CRVAL2': 0,
+        'CTYPE4': 'HPLN-TAN',
+        'CUNIT4': 'arcsec',
+        'CDELT4': 5,
+        'CRPIX4': 5,
+        'CRVAL4': 0,
 
-        'CTYPE1': 'TIME    ',
-        'CUNIT1': 'min',
-        'CDELT1': 0.4,
-        'CRPIX1': 0,
-        'CRVAL1': 0,
+        'DATEREF': "2020-01-01T00:00:00"
```

Comment:
In theory could this also be `'DATE-OBS`?  Or is `'DATEREF` the key that astropy WCS recognises?

Reply:
I think dateref is something slightly different. I think this is a hold over from a SpectralCoord bug.